demo/app.js

```diff
@@ -655,6 +657,7 @@ const Demo = React.createClass({
               Footer Content
             </div>
           )}
+          footerStyle={{ padding: '40px 20px' }}
```

Comment:
In the demo app file we just inline it.  Not sure why we went that way but I think it was more for ease of setting up demos.  Everywhere else though we always do the style object/function.